client/src/utils/localSsot/parameters.js

```diff
@@ -0,0 +1,291 @@
+/**
+ * @category Client
+ * @module
+ */
+import getRpaFunctionalitiesObject from '../rpaFunctionality/functionalities';
+import { getAttributeObjectForActivity } from './attributes';
+
+const PARAMETER_STORAGE_PATH = 'parameterLocalStorage';
+
+/**
+ * @description Creates a parameter object in the parameter session storage
+ * @param {Array} localParameterStorage An Array containing the current parameter objects
+ * @param {string} robotId The id of the selected robot
+ * @param {string} activityId The id of the selected activity
+ * @param {String} application The name of the rpa application the activity has selected
+ * @param {String} task The name of the rpa task the activity has selected
+ * @returns {Object} Newly created parameter object
+ */
+const createParameterObject = (
+  localParameterStorage,
+  robotId,
+  activityId,
+  application,
+  task
+) => {
+  const rpaFunctionalitiesObject = getRpaFunctionalitiesObject(
+    application,
+    task
+  );
+
+  const rpaParameters = [];
+  if (rpaFunctionalitiesObject && rpaFunctionalitiesObject.inputVars) {
+    rpaFunctionalitiesObject.inputVars.forEach((element) => {
+      const elementCopy = element;
+      elementCopy.value = '';
+      rpaParameters.push(elementCopy);
+    });
+  }
+
+  const newParameterObject = {
+    activityId,
+    outputVariable:
+      rpaFunctionalitiesObject && rpaFunctionalitiesObject.outputValue
```

Comment:
not sure if it should be initialised with undefined. I would rather have an error thrown or so when either value is not defined instead of setting the value undefined. Please check if that is an option

Reply:
I just changed it so that the new parameter object gets created without an outputVariable property if the condition is not met. ✅